async-polling/async-polling.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for AsyncPolling
+// Project: https://github.com/cGuille/async-polling
+// Definitions by: Zlatko Andonovski <https://github.com/Goldsmith42/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "async-polling" {
+    module AsyncPolling {
```

Comment:
Will do, thanks. My brain automatically reverted to pre-1.5 and TSLint failed to catch it for some reason.